crossbeam-channel/src/flavors/mod.rs

```diff
@@ -9,9 +9,9 @@
 //! 5. `tick` - Channel that delivers messages periodically.
 //! 6. `zero` - Zero-capacity channel.
 
-pub mod array;
-pub mod at;
-pub mod list;
-pub mod never;
-pub mod tick;
-pub mod zero;
+pub(crate) mod array;
```

Comment:
Hmm, instead of this, can we make the `flavors` module `pub(crate)`?

Reply:
The parent modules are already private, but rustc says they are still unreachable `pub` because we can re-export those modules.